src/transformStateWithClones.js

```diff
@@ -8,6 +8,32 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const newObj = { ...state };
+  const resArr = [];
+
+  for (const { type, extraData, keysToRemove } of actions) {
```

Comment:
```suggestion
  for (action of actions) {
  const { 
    type, 
    extraData, 
    keysToRemove, 
  } = action
```
Your code should be not just short, but also understandable and descriptive